waiter/src/waiter/handler.clj

```diff
@@ -715,21 +724,29 @@
                            (service-description->service-id))]
         (counters/inc! (metrics/waiter-counter "auto-run-as-requester" "form-render"))
         (meters/mark! (metrics/waiter-meter "auto-run-as-requester" "form-render"))
-        {:body (template/eval (slurp (io/resource "web/consent.html"))
-                              {:auth-user auth-user
-                               :consent-expiry-days consent-expiry-days
-                               :service-description-template service-description-template
-                               :service-id service-id
-                               :target-url (str (name (utils/request->scheme request)) "://" host-header "/" path
-                                                (when (not (str/blank? query-string)) (str "?" query-string)))
-                               :token token})
+        {:body (render-consent-template
+                 {:auth-user auth-user
+                  :consent-expiry-days consent-expiry-days
+                  :service-description-template service-description-template
+                  :service-id service-id
+                  :target-url (str (name (utils/request->scheme request)) "://" host-header "/" path
+                                   (when (not (str/blank? query-string)) (str "?" query-string)))
+                  :token token})
          :headers {"content-type" "text/html"}
          :status 200}))
     (catch Exception ex
       (counters/inc! (metrics/waiter-counter "auto-run-as-requester" "form-error"))
       (meters/mark! (metrics/waiter-meter "auto-run-as-requester" "form-error"))
       (utils/exception->response ex request))))
 
+(let [html-fn (template/fn [{:keys [cid host hostname message port support-info timestamp]}]
+                           (slurp (io/resource "web/welcome.html")))]
+  (defn- render-welcome-html "Renders welcome html" [context] (html-fn context)))
+
+(let [text-fn (template/fn [{:keys [cid host hostname message port support-info timestamp]}]
+                           (slurp (io/resource "web/welcome.txt")))]
+  (defn- render-welcome-text "Renders welcome text" [context] (text-fn context)))
```

Comment:
These `defn`s where everything is on the same line look odd to me

Reply:
Done.